src/olympia/devhub/tests/test_views_subscription.py

```diff
@@ -168,44 +172,46 @@ def test_shows_confirmation_after_payment(self, retrieve_mock):
 
     @mock.patch("olympia.devhub.views.retrieve_stripe_checkout_session")
     def test_shows_confirmation_with_new_version_number(self, retrieve_mock):
-        self.addon.current_version.update(version='12.3')
+        self.addon.current_version.update(version="12.3")
         stripe_session_id = "some session id"
         retrieve_mock.return_value = mock.MagicMock(id=stripe_session_id)
         self.subscription.update(
             stripe_session_id=stripe_session_id,
-            payment_completed_at=datetime.datetime.now(),
+            checkout_completed_at=datetime.datetime.now(),
         )
 
         response = self.client.get(self.url)
         assert b"You're done!" in response.content
         assert b"<strong>12.3.1-signed</strong>" in response.content
-        assert b'will be published as Verified' in response.content
+        assert b"will be published as Verified" in response.content
```

Comment:
(I applied black on the file after the renaming. It's a new file that I initially created so black was already applied on it before)